### PR TITLE
Pipelines 1.14.0 cli download link version bump

### PR DESCRIPTION
**Pull request by SNiemann15**
This is for the upcoming Pipelines 1.14.0 GA 

Preview https://deploy-preview-31552--osdocs.netlify.app/openshift-enterprise/latest/cli_reference/tkn_cli/installing-tkn.html

@ppitonak Can you QE review please. Thanks!

cc @Preeticp @

### CODE REVIEW

**Comment by ppitonak** [comment]
IMHO you could update also Windows and MacOS links in this PR so it's easier to follow. WDYT @Preeticp

**Comment by SNiemann15** [comment]
> IMHO you could update also Windows and MacOS links in this PR so it's easier to follow. WDYT @Preeticp

@ppitonak I'm on PTO tomorrow and Friday but if its ok for Preeti I can do it now. Is it also 0.17.2 for Windows and MacOS?

**Comment by ppitonak** [comment]
Yes, the same version on all platforms and operating systems

**Comment by SNiemann15** [comment]
@ppitonak I've changed the versions for Windows and Macos and also changed the version in RPM as discussed.

File: modules/op-installing-tkn-on-linux-using-rpm.adoc [hunk 1]
```diff
@@ -47,7 +47,7 @@ For {op-system-base-full} version 8, you can install the {pipelines-title} CLI (
 +
 [source,terminal]
 ----
-# subscription-manager repos --enable="pipelines-1.3-for-rhel-8-x86_64-rpms"
+# subscription-manager repos --enable="pipelines-1.4-for-rhel-8-x86_64-rpms"
```
Comment (ppitonak) [inline modules/op-installing-tkn-on-linux-using-rpm.adoc:50]: @VeereshAradhya can you confirm that the repo is correct (i.e. there was no change in naming conventions for GA)

**Review by ppitonak — APPROVED** [review]
/lgtm

File: modules/op-installing-tkn-on-linux-using-rpm.adoc (hunk 1 above)
Comment (Preeticp) [reply modules/op-installing-tkn-on-linux-using-rpm.adoc:52]: @VeereshAradhya do we have a confirmation on this?

Comment (VeereshAradhya) [reply modules/op-installing-tkn-on-linux-using-rpm.adoc:52]: yes.. verified.. looks good

File: modules/op-installing-tkn-on-linux-using-rpm.adoc [hunk 2]
```diff
@@ -45,9 +45,25 @@ For {op-system-base-full} version 8, you can install the {pipelines-title} CLI (
 
 . Enable the repositories required by {pipelines-title}:
 +
+Linux (x86_64, amd64)
```
Comment (Preeticp) [inline modules/op-installing-tkn-on-linux-using-rpm.adoc:48]: @SNiemann15 as you mentioned, bullets would definitely make it look much better. Do add + on an empty line before the code snippet to ensure they are aligned properly.

**Pull request merged**